### PR TITLE
[data-network] made usable

### PR DESCRIPTION
**Pull request by guicho271828**

### CODE REVIEW

File: data-network/generator/generator.py [hunk 1]
```diff
@@ -537,43 +537,17 @@ def _natural_keys(self, text):
...
-                    num_data_items + 2, num_data_items * 3, num_data_items // 2
-                ):
+parser = argparse.ArgumentParser()
+parser.add_argument("data", type=int, help="the number of data items")
+parser.add_argument("layers", type=int, help="the number of layers, must be smaller than the number of data items")
+parser.add_argument("scripts", type=int, help="the number of scripts, must be larger or equal than [number of data items]-2")
+parser.add_argument("module", choices=["ring-network","small-network","tiny-network"],
+                    metavar="module",
+                    help='A module describing a network, one of %(choices)s. See example files in the source code directory.')
+parser.add_argument("--seed", type=int, default=42, help="random seed")
```
Comment (jendrikseipp) [inline data-network/generator/generator.py:549]: I think the generator shouldn't become deterministic when no seed is given. IIRC, you simply need to change 42 to None. This should be sufficient for the Python random module. Not sure about the numpy random module.

**Review by jendrikseipp — CHANGES_REQUESTED** [review]

File: data-network/generator/generator.py (hunk 1 above)
Comment (guicho271828) [reply data-network/generator/generator.py:549]: oh that is right

**Comment by guicho271828** [comment]
fixed

**Comment by jendrikseipp** [comment]
Looks good. Can you fix the failing "make test"? Then we can merge this.

**Comment by jendrikseipp** [comment]
Thanks!

**Pull request merged**